### ISSUE TITLE
Space when a terminal tab has focus should move focus to the instance

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#122025]
Repro:

1. Create a terminal
2. Click the tab
3. Press space, the actual terminal should be focused

**Comment by Tyriar** [comment]
It's cmd+down on mac, just like in the explorer. Enter already works. Closing as it's already consistent
